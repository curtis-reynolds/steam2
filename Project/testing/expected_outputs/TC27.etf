02_valid___________FS_000000.00
00_admin___________AA_000000.00
