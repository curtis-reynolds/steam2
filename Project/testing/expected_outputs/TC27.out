Login successful.
User deleted successfully.
